x-pack/test/detection_engine_api_integration/security_and_spaces/group10/risk_engine/init_and_status_apis.ts

```diff
@@ -29,6 +31,16 @@ export default ({ getService }: FtrProviderContext) => {
   const log = getService('log');
 
   describe('Risk Engine', () => {
+    beforeEach(async () => {
+      await cleanRiskEngineConfig({ kibanaServer });
+      await deleteRiskEngineTask({ es, log });
+      await deleteAllRiskScores(log, es);
+      await clearTransforms({
+        es,
+        log,
+      });
+    });
```

Comment:
We should abstract this to a function named `cleanRiskEngine({ kibanaServer, es, log })`